### PR TITLE
Kanban Project Final Review

### PR DESCRIPTION
**Pull request by Hidar37**
- Project All **functionality** working.
- **Linter** Error Fixed.
- **Involvement** API use.
- **Style** add according to wire frame.
- **ES6** syntax use.
- **Kanban** project use for this project.
- **Zoom** video link add to readme file.
**NOTE:**
_I have try to add the live demo link but i got this error and i could not find a way out to solved please cooperate. Here is the screen short of the problem:_
hi
**Important:**
_My coding partner had a very important work to do that is way he could not made to the zoom call you can ask him also._

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has linter errors. Kindly look into the error and fix it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

linter

**Comment by Hidar37** [comment]
@Qoosim 
Hello Dear sir 
I have given the reason that why i have this Issus.
please help.

File: src/style/style.css [hunk 1]
```diff
@@ -0,0 +1,306 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: cursive;
+}
+
+.nav {
+  background-color: #ffe200;
```
Comment (Whoistolu) [inline src/style/style.css:9]: - [ ] [OPTIONAL] Great job guys 👍🏽 However, for a better UI(User Interface), I would suggest that you use a yellow colour with a higher gradient. Something like `#7e7004`. Currently, the whole landing page seems too bright.

**Review by Whoistolu — APPROVED** [review]
### Hi Team,
@Hidar37 and @habasm 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [OPTIONAL] You had those issues because you tried deploying with Github pages. in as much as it is not a requirement for this specific capstone. You can always deploy the project with netlify, render or heroku.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**